xml/Microsoft.VisualBasic.FileIO/TextFieldParser.xml

```diff
@@ -1455,7 +1455,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Releases resources used by the <see cref="T:Microsoft.VisualBasic.FileIO.TextFieldParser" /> object.</summary>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>  
          <format type="text/markdown"><![CDATA[  

This member is an explicit interface member implementation. It can be used only when the `TextFieldParser` instance is cast to an <xref:System.IDisposable> interface. The recommended alternative is to call the <xref:Microsoft.VisualBasic.FileIO.TextFieldParser.Dispose%2A> method.

         ]]></format>
         </remarks>
```